### DISCUSSION TITLE
How to customize inactive tabs?

### DISCUSSION THREAD

**Discussion by KleoPadre** [Akifyss/obsidian-border#230] [Q&A] [answered]
My inactive tabs are unreadable. I couldn't find where to customize them, I only found the active tabs setting. Same problem with the settings button
![Снимок экрана 2024-04-03 в 11 36 39](https://github.com/Akifyss/obsidian-border/assets/88144121/dabdf6a2-2449-48a1-9232-238519eab757)
![Снимок экрана 2024-04-04 в 09 26 35](https://github.com/Akifyss/obsidian-border/assets/88144121/1301f2ba-f6e2-488a-8032-3db171921774)

**Comment by Akifyss** [comment, marked as answer]
I'm sorry for forgetting to answer your question. I don't know if you have resolved it, but you should be able to customize it at the following location:
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/f3ed20c1-b25b-4d8e-8b2d-de99e66f1153)
